docs/framework/dataframe.mdx

```diff
@@ -2,97 +2,213 @@
 title: "Dataframe"
 ---
 
-**writer framework places the dataframe at the core of the application**. This is a great way for modeling a complex and massive data system.
-it offers components as `dataframe` to manipulate dataframes. These components allow you to visualize and interact with dataframes.
-
-| compatibility      | dataframe                             |
-|--------------------|---------------------------------------|
-| `pandas.DataFrame` | x                                     |
-| `polar.DataFrame`  | x                                     |
-| `list of records`  | x (with `EditableDataframe`)          |
-
-## Use a dataframe
-
-**a dataframe is simply added to the state**. A component like `dataframe` will be able to display it.
+**Writer Framework has two objects with the name _DataFrame_:**
+
+1. **Code Dataframe:** In code that you write for a Writer Framework application, `DataFrame` refers to a **_data structure_** that stores data in rows and columns in a way similar to a spreadsheet or SQL table.
+2. **UI DataFrame:** In Writer Framework's UI, "DataFrame" refers to a **_user interface component_** that displays data in rows and columns in a way similar to a spreadsheet or SQL table. 
+
+To present a data table in Writer Framework, you create a `DataFrame` data structure in your code and then bind it to a UI Dataframe.
+
+
+## Displaying a static DataFrame
```

Comment:
Can we please briefly discuss the options in the DataFrame component (e.g. sorting, downloading, searching) and link to the component documentation.